lib/phoenix_ecto/plug.ex

```diff
@@ -23,3 +23,40 @@ unless Ecto.SubQueryError in excluded_exceptions do
     end
   end
 end
+
+unless Phoenix.Ecto.PendingMigrationError in excluded_exceptions do
+  defimpl Plug.Exception, for: Phoenix.Ecto.PendingMigrationError do
+    def status(_error), do: 404
+
+    def actions(%{repo: repo}),
+      do: [
+        %{
+          label: "Run migrations for repo",
+          handler: {__MODULE__, :migrate, [repo]}
+        }
+      ]
+
+    def migrate(repo) do
+      if Process.whereis(repo),
```

Comment:
Is there a reason why you added the `whereis` check here? The only way we can get to this route is if the repo exists already, no?

Reply:
Yep, don't remember why I added this here. Will remove it 😄

Reply:
[fixed here](https://github.com/phoenixframework/phoenix_ecto/compare/846278a787c3300abfadce9a9d8097da9131afe2..5083d99f7f515143f5d00420d620f5491833597f)